esmvalcore/preprocessor/_time.py

```diff
@@ -432,33 +432,63 @@ def anomalies(cube, period):
         Available periods: 'full', 'season', 'seasonal', 'monthly', 'month',
         'mon', 'daily', 'day'
 
+    standardize: bool, optional
+        If True standardized anomalies are calculated
+
+
     Returns
     -------
     iris.cube.Cube
-        Monthly statistics cube
+        Anomalies cube
     """
     reference = climate_statistics(cube, period=period)
     if period in ['full']:
-        return cube - reference
+        cube = cube - reference
+        if standardize:
+            cube_stddev = climate_statistics(
+                cube, operator='std_dev', period=period)
+            return cube / cube_stddev
+        else:
```

Comment:
This `else` can be removed

Reply:
Are you sure? One of the checkers also told me to remove it, but I think we need it. The code below line 452 should not be executed in the case that `period='full'`. `_get_period_coord` will raise ValueError, since `full` is not accepted as argument to `period`.

Reply:
The linter is right, becaues if `standardize` evaluates to `True`, you will leave the function on line 450. If it's false, it will continue. Maybe it would be more clear to write
```python
    if period == 'full':
        cube = cube - reference
        if standardize:
            cube_stddev = climate_statistics(
                cube, operator='std_dev', period=period)
            cube = cube / cube_stddev
        return cube

Reply:
Yes! Now I get it. It suggests removing solely the `else` not the `else`-block. Your suggestion is much clearer I will go with that.